packages/eslint-plugin/src/resolve-workspace-rules.ts

```diff
@@ -11,7 +12,9 @@ export const workspaceRules = ((): ESLintRules => {
     return {};
   }
   // Register `tools/eslint-rules` for TS transpilation
-  const registrationCleanup = registerTsProject(WORKSPACE_PLUGIN_DIR);
+  const registrationCleanup = registerTsProject(
```

Comment:
Can we use `joinPathFragments` instead of `join` here?

Reply:
No, this is not part of a generator and we are referring to paths on the real disk. They need to be what the OS expects, and may not be on the `C:` drive on windows.